modules/nw-osp-specify-floating-ip.adoc

```diff
@@ -0,0 +1,64 @@
+// Modules included in the following assemblies:
+//
+// * networking/load-balancing-openstack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-osp-specify-floating-ip_{context}"]
+= Specifying a floating IP address in the Ingress Controller
+
+By default, a floating IP address gets randomly assigned to your {product-title} cluster on {rh-openstack-first} upon deployment. This floating IP address is associated with your Ingress port.
```

Comment:
@EmilienM, is it true that a _floating IP_ address gets assigned by default?  [The API godoc](https://github.com/openshift/cluster-ingress-operator/blob/8be174945db79b0660bd0fd21ed279ab7b6aafbf/vendor/github.com/openshift/api/operator/v1/types_ingress.go#L685-L692) says, "When not specified, an IP address will be assigned randomly by the OpenStack cloud provider", and I thought OpenStack didn't use a floatingip in this case (otherwise I would have asked for the godoc to say that "a _floating IP_ address will be assigned randomly" for the default case).

Reply:
Yes. If you don't specify a floating IP, CCM (when creating the Load Balancer in OpenStack Octavia) will create a floating IP from the External Network configured in CCM.

> I thought OpenStack didn't use a floatingip in this case (otherwise I would have asked for the godoc to say that "a floating IP address will be assigned randomly" for the default case).

Since the string is optional (and therefore not set by default), we say that "when not specified, an IP address will be assigned randomly by the OpenStack cloud provider.". We could have added "(the default)" after "when not specified" but I think it's ok.